public/views/index-template.html

```diff
@@ -199,7 +199,7 @@
         </p>
         <p>
           1. This could be caused by your reverse proxy settings.<br /><br />
-          2. If you host grafana under subpath make sure your grafana.ini root_url setting includes subpath<br />
+          2. If you host grafana under subpath make sure your grafana.ini root_url setting includes subpath. If not using a reverse proxy make sure to set serve_from_sub_path to true.
```

Comment:
Is there a reason why the `<br />` at the end of the line is removed? Seems that other list items end with to `<br />`.

Reply:
good catch; it was by mistake